x-pack/legacy/plugins/ingest/server/libs/policy.ts

```diff
@@ -258,7 +263,36 @@ export class PolicyLib {
       shared_id: newPolicyInfo.shared_id,
     };
   }
-
+  public async ensureDefaultPolicy() {
+    try {
+      await this.adapter.get(DEFAULT_POLICY_ID);
+    } catch (err) {
+      if (!err.isBoom || err.output.statusCode !== 404) {
+        throw err;
+      }
+      const info = this.libs.framework.info;
+      if (info === null) {
+        throw new Error('Could not get version information about Kibana from xpack');
+      }
+      const newDefaultPolicy: NewPolicyFile = {
+        name: 'Default Policy',
+        description: 'default policy create by kibana (not possible to delete)',
+        status: 'active',
+        monitoring_enabled: true,
```

Comment:
@ph or @urso Should the default policy have monitoring on by default? I would think yes but just want to make sure.

Reply:
I am going to merge it like this, do not hesitate to comment later, if we want to change this.